lint-configs/template-lint-rules/no-at-class.mjs

```diff
@@ -0,0 +1,27 @@
+import { Rule } from "ember-template-lint";
+
+const AFFECTED_COMPONENTS = ["DButton"];
```

Comment:
A couple extra from all-the*

```suggestion
const AFFECTED_COMPONENTS = [
  "DButton",
  "DModal",
  "TableHeaderToggle",
  "Textarea",
  "TextArea",
];
```

(yes, those are two different textarea components 😂)